Components/Filter/Filter.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://iconica.netlify.app/Components.css">
+    <link rel="stylesheet" href="/Components/Filter/Filter.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="navigation nav-color">
+        <div class="navigation_Title"> Iconica Store</div>
+        <div class="searchbar"><input type="text" placeholder="Search"></div>
+        <div class="navigation_Interaction">
+            <a class="navigation__login" href="/Components/Login and Signup/Login.html">Login</a>
```

Comment:
Space b/w Folder names is not recommended

Reply:
noted